facade/src/main/scala/reactST/reactTable/TableMaker.scala

```diff
@@ -8,7 +8,7 @@ import react.common.Css
 import react.virtuoso.Virtuoso
 import reactST.reactTable.anon.Data
 import reactST.reactTable.mod.ColumnInterfaceBasedOnValue._
-import reactST.reactTable.mod._
+import reactST.reactTable.mod.{ ^ => _, _ }
```

Comment:
The newer scalablytyped also changed what gets imported with this import to include the `^` symbols in scalajs-react. This resulted in duplicate symbols errors. Clients of the library may, unfortunately, need to care about this.